server/dto/session_stats.go

```diff
@@ -4,4 +4,5 @@ type SessionStats struct {
 	BytesSent     int    `json:"bytes_sent"`
 	BytesReceived int    `json:"bytes_received"`
 	ProviderID    string `json:"provider_id"`
+	ClientCountry string `json:"client_country"`
```

Comment:
why not just "country" ? client/server terms should disappear in long term.

Reply:
Since these stats are for session, it's good to indicate which country it is, since there are 2 countries in session.
Just not `ClientCountry`, but `ConsumerCountry`.

Reply:
We have Providers and Consumer (server&client is software pieces)

Reply:
renamed to ConsumerCountry